apps/studio/src/components/NotificationManager.vue

```diff
@@ -20,21 +22,45 @@ export default Vue.extend({
           Noty.button('Close', 'btn btn-flat', () => Noty.closeAll('upsell')),
           Noty.button('Get Started', 'btn btn-primary', () => window.main.openExternally('https://docs.beekeeperstudio.io/docs/upgrading-from-the-community-edition'))
         ]
-      })
+      }
     }
   },
-  mounted() {
-    if (this.$store.getters.isCommunity) {
-      const today = new Date()
-      const upgradeSuggested = SmartLocalStorage.getDate('ultimate-upsell')
-      const lastWeek = new Date(today.getTime() - (28 * 24 * 60 * 60 * 1000))
-      if (!upgradeSuggested || upgradeSuggested < lastWeek) {
-        setTimeout(() => {
-          this.upsellNotification.show()
-          SmartLocalStorage.setDate('ultimate-upsell', today)
-        }, (1000 * 60 * 5)) // 5 minutes
+  computed: {
+    ...mapGetters({
+      'isCommunity': 'isCommunity',
+    })
+  },
+  watch: {
+    isCommunity() {
+      this.initNotifyInterval()
+    }
+  },
+  methods: {
+    initNotifyInterval() {
+      const intervalTime = 1000 * 60 * 60 * 3
+      if (this.notificationInterval) {
+        this.notificationInterval = null
+        clearInterval(this.notificationInterval)
+      }
+      if (this.timeoutID) {
+        this.timeoutID = null
+        clearTimeout(this.timeoutID)
```

Comment:
Could be wrong here, but if you're setting the `notificationInterval` and `timeoutID` before clearing them, then clearing them does nothing, right?

Reply:
I think you may be right here. Oops. I switched the logic to clear the interval/timeout first, then set the data to null so hopefully the madness has stopped